reference/shaders-msl/vulkan/frag/basic.multiview.nocompat.vk.frag

```diff
@@ -0,0 +1,25 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct main0_out
+{
+    float4 FragColor [[color(0)]];
+};
+
+struct main0_in
+{
+    float4 vColor [[user(locn0)]];
+    float2 vTex [[user(locn1)]];
+};
+
+fragment main0_out main0(main0_in in [[stage_in]], constant uint* spvViewMask [[buffer(24)]], texture2d<float> uTex [[texture(0)]], sampler uTexSmplr [[sampler(0)]], uint gl_ViewIndex [[render_target_array_index]])
+{
+    main0_out out = {};
+    if (!(*spvViewMask & (1 << gl_ViewIndex)))
```

Comment:
This is really bad, and we have to avoid this. This forces late-Z for everything, and I think this is completely avoidable if we pass down the view mask as proposed above.

Reply:
The alternative is to set w = -1 in vertex shaders which should mask rasterization if it's not possible to do it like I proposed above.

Reply:
I've decided to split the subpass into multiple passes in case there's a gap in the mask.